### PR TITLE
Add doc related to sumWeights <= 1 case when computing variance

### PR DESCRIPTION
**Pull request by vthiery**
When the sum of weights equals 1, the unbiased estimator used to compute the variance is ill-defined. This PR adds some doc to inform the user that a biased estimator should be used instead

### CODE REVIEW

File: stat/stat.go [hunk 1]
```diff
@@ -1076,6 +1076,8 @@ func empiricalQuantile(p float64, x, weights []float64, sumWeights float64) floa
 // Skew computes the skewness of the sample data.
 // If weights is nil then all of the weights are 1. If weights is not nil, then
 // len(x) must equal len(weights).
+// Note that in case of weights summing to less than (or equal) to 1,
```
Comment (kortschak) [inline stat/stat.go]: Please remove the parentheses here and below.

I would prefer something like "`// When weights sum to 1 or less, a biased variance estimator should be used.`"

Comment (vthiery) [reply stat/stat.go]: That is indeed clearer, thanks!

**Comment by btracey** [comment]
It looks like they (or we) changed a setting where all reviewers have to approve? Not sure we want our policy to be that way.

**Comment by kortschak** [comment]
>It looks like they (or we) changed a setting where all reviewers have to approve? Not sure we want our policy to be that way.

No, I'm pretty sure that is there's a request for changes that's always been blocking. If worst comes to worst, another person can always dismiss the review.

**Pull request merged**